pkg/controllers/job/job_controller_actions.go

```diff
@@ -458,6 +474,69 @@ func (cc *jobcontroller) syncJob(jobInfo *apis.JobInfo, updateStatus state.Updat
 	return nil
 }
 
+func (cc *jobcontroller) waitDependsOnTaskMeetCondition(taskName string, podToCreateEachTask []*v1.Pod, job *batch.Job) {
+	taskIndex := jobhelpers.GetTasklndexUnderJob(taskName, job)
+	if job.Spec.Tasks[taskIndex].DependsOn != nil {
+		dependsOn := *job.Spec.Tasks[taskIndex].DependsOn
+		if len(dependsOn.Name) > 1 && dependsOn.Iteration == batch.IterationAny {
+			wait.PollInfinite(100*time.Millisecond, func() (bool, error) {
+				for _, task := range dependsOn.Name {
+					if cc.isDependsOnPodsReady(task, job) {
+						return true, nil
+					}
+				}
+				return false, nil
+			})
+		} else {
+			for _, dependsOnTask := range dependsOn.Name {
+				wait.PollInfinite(100*time.Millisecond, func() (bool, error) {
```

Comment:
it's better to make 100ms configurable. At least define a const for it if no time :)

Reply:
I implemented configurable parameters. But I don't know if the implementation is elegant, especially the process of variable transmission. In addition, the naming of variables is also very long. If you have any comments, please let me know, thank you!